jena-geosparql/src/main/java/org/apache/jena/geosparql/implementation/index/CacheConfiguration.java

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Wrong header.

It should be the 
"""
Licensed to the Apache Software Foundation (ASF) under one
or more contributor license agreements.
""" 
header. 

See adjacent files for the text.